phpunit.xml

```diff
@@ -22,7 +22,7 @@
         <server name="BCRYPT_ROUNDS" value="4"/>
         <server name="CACHE_DRIVER" value="array"/>
         <server name="DB_CONNECTION" value="mysql"/>
-        <server name="DB_DATABASE" value="wave_testing"/>
+        <server name="DB_DATABASE" value="tests"/>
```

Comment:
```suggestion
        <server name="DB_DATABASE" value="wave_testing"/>
```
Wave's GitHub workflow starts a MySQL server with a database named `wave_testing`. Changing it will cause the GitHub workflow to stop working. I highly recommend changing this back.

https://github.com/thedevdojo/wave/blob/d71d14fe1746e244103fe79ed31e6da1ebdd6875/.github/workflows/tests.yaml#L81

Reply:
My fault!!